src/pages/dashboard/components/Profile/control/index.tsx

```diff
@@ -53,6 +58,28 @@ export const Profile = () => {
           <InfoRow color="text-gray-200" title="net worth">
             <Price color="text-white" value={balance} />
           </InfoRow>
+
+          {self && !self.discordId && (
+            <InfoRow>
+              <Link href={DISCORD_OAUTH_URL}>
+                <a>
```

Comment:
I think it's better to create a standalone component named like `LinkDiscordButton` or something similar, as we need to use it in multiple touchpoints